### PR TITLE
Updated library flickity to 2.0.9 for compatibility with WordPress 4.8.0.

### PR DESCRIPTION
**Pull request by ChristophKnopf**

### CODE REVIEW

File: plugin.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Gallerya
-  Version: 1.9.4
+  Version: 2.0.9
```
Comment (fabianmarz) [inline plugin.php]: Why this massive version plugin version change? Updating the dependencies in minor versions should not update the plugin in a major version. Please see [semver.org](http://semver.org) for how to versioning.

File: bower.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     },
     "lightgallery": {
       "main": [
-        "./dist/js/lightgallery.min.js",
+        "./dist/js/lightgallery.js",
```
Comment (fabianmarz) [inline bower.json]: Why did you change this to the non minified version?

File: plugin.php (hunk 1 above)
Comment (ChristophKnopf) [inline plugin.php]: I accidentally took the version from flickity which was on my mind.

Comment (ChristophKnopf) [inline plugin.php]: I accidentally took the version from flickity which was on my mind.

File: bower.json (hunk 2 above)
Comment (ChristophKnopf) [inline bower.json]: uglify-js messes up with the minified version of lightgallery as can be read here [Uncaught TypeError: i.data is not a function](https://github.com/sachinchoolur/lightGallery/issues/458).
This may be also be fixed with an lightgallery update to 1.5.0.

**Comment by ChristophKnopf** [comment]
to 1. I wondered a little about that but did not touch because of '>' . I have to read about best practices for this.

to 2. There have been no changes to lightgallery yet. I did not know about being so specific in library updates commit messages, I will clean up.

**Comment by ChristophKnopf** [comment]
@fabianmarz and @sun some information for this fixup.
I did some changes as you can see. Did a `npm init` to update the package.json and changed deprecated licences propertie by the way.

For the dependencies in bower.json, especially specifying version ranges, I decided to use version has to match exactly as this version is tested and productive. Another option is to use `>=` .
